mne/io/artemis123/artemis123.py

```diff
@@ -168,6 +168,10 @@ def _get_artemis123_info(fname):
              'logno': i + 1, 'scanno': i + 1, 'range': 1.0,
              'unit_mul': FIFF.FIFF_UNITM_NONE,
              'coord_frame': FIFF.FIFFV_COORD_DEVICE}
+        # REF_018 has a zero cal which can cause problems. Let's set it to
+        # a value of another ref channel to make writers/readers happy.
+        if t['cal'] == 0:
+            t['cal'] = 4.716e-10
```

Comment:
cc @bloyl okay with you? Without this change we get `invalid division` warnings when writing because we divide out by `cals`, which was causing division by zero.